### ISSUE TITLE
[Bug] Trying to Switch Page Layouts

### ISSUE THREAD

**Issue by mary-grace** [mattstratton/castanet#233]
### Expected behavior
Trying to switch page layout from `row` to `grid` -- I expect to be able to toggle between these two using those exact phrases in the config.toml `site_layout = ""` line.

### Actual behavior
I get an error message when running it on my local machine:
[Screenshot](https://www.dropbox.com/s/j5dqiczxnihbrbg/Screenshot%202019-11-22%2014.41.05.png?dl=0)
```render of "home" failed: execute of template failed: template: index.html:5:8: executing "main" at <partial "grid.html" .>: error calling partial: "/Users/duck/repos/community-pulse-source/themes/castanet/layouts/partials/grid.html:104:33": execute of template failed: template: partials/grid.html:104:33: executing "partials/grid.html" at <.Permalink>: can't evaluate field Permalink in type *page.Pager
failed to render pages: render of "home" failed: execute of template failed: template: index.html:5:8: executing "main" at <partial "grid.html" .>: error calling partial: "/Users/duck/repos/community-pulse-source/themes/castanet/layouts/partials/grid.html:104:33": execute of template failed: template: partials/grid.html:104:33: executing "partials/grid.html" at <.Permalink>: can't evaluate field Permalink in type *page.Pager
```

This happens whether `jumbotron` is true or false.

### Reproduction Steps
Switch config.toml `site_layout = ""` line from `row` to `grid`

### Operating System and Version
Mac OS 10.14.6

### Web Browser and Version
Chrome 78.0.3904.87

### Hugo Version
v0.59.1/extended darwin/amd64

**Comment by mattstratton** [comment]
I was having the same issue when trying to upgrade arresteddevops.com to the latest Castanet, so it's the same bug I think. So the good news is, I can see it replicate in your site too. I'll work on this today.

**Comment by mattstratton** [comment]
I fixed this in #235; after I merge it I'll cut a release and that should fix it for you.

**Comment by mattstratton** [comment]
If you upgrade Castanet to [1.1.20](https://github.com/mattstratton/castanet/releases/tag/1.1.20) it should work now!
